<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Setup</class>
 <widget class="QDialog" name="Setup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>750</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Place Battleships</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/pictures/icon.png</normaloff>:/pictures/icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QFrame" name="gridFrame">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>651</width>
     <height>281</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(34,139,34);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="lineWidth">
    <number>5</number>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <property name="spacing">
     <number>10</number>
    </property>
    <item row="3" column="3">
     <widget class="QSpinBox" name="carrier_spinBox">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(34,139,34);</string>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>10</number>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QSpinBox" name="battleship_spinBox">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(34,139,34);</string>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>10</number>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QSpinBox" name="battleship_spinBox_2">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(34,139,34);</string>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>10</number>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Maximum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>26</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="carrierlength_label">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 700 10pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;5&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QSpinBox" name="carrier_spinBox_2">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(34,139,34);</string>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>10</number>
      </property>
     </widget>
    </item>
    <item row="4" column="5" alignment="Qt::AlignHCenter">
     <widget class="QComboBox" name="battleship_comboBox">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Segoe UI&quot;;
background-color: rgb(34,139,34);
color: rgb(255, 255, 255);</string>
      </property>
      <item>
       <property name="text">
        <string>Horizontal</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Vertical</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QLabel" name="patrolboatlength_label">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 700 10pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;2&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="7" column="4">
     <widget class="QSpinBox" name="patrolboat_spinBox_2">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(34,139,34);</string>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>10</number>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="ship_label">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 700 18pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; text-decoration: underline;&quot;&gt;Ship&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="5" column="5" alignment="Qt::AlignHCenter">
     <widget class="QComboBox" name="destroyer_comboBox">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Segoe UI&quot;;
background-color: rgb(34,139,34);
color: rgb(255, 255, 255);</string>
      </property>
      <item>
       <property name="text">
        <string>Horizontal</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Vertical</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QLabel" name="orientation_label">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 700 18pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; text-decoration: underline;&quot;&gt;Orientation&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLabel" name="Submarine_label">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 700 12pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Submarine:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="5" alignment="Qt::AlignHCenter">
     <widget class="QComboBox" name="carrier_comboBox">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Segoe UI&quot;;
background-color: rgb(34,139,34);
color: rgb(255, 255, 255);</string>
      </property>
      <item>
       <property name="text">
        <string>Horizontal</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Vertical</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="7" column="3">
     <widget class="QSpinBox" name="patrolboat_spinBox">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(34,139,34);</string>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>10</number>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Maximum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>0</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="6" column="3">
     <widget class="QSpinBox" name="submarine_spinBox">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(34,139,34);</string>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>10</number>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QLabel" name="submarinelength_label">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 700 10pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;3&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QSpinBox" name="destroyer_spinBox">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(34,139,34);</string>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>10</number>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLabel" name="patrolboat_label">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 700 12pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Patrol Boat:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="battleship_label">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 700 12pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Battleship:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QLabel" name="destroyerlength_label">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 700 10pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;3&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="6" column="4">
     <widget class="QSpinBox" name="submarine_spinBox_2">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(34,139,34);</string>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>10</number>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLabel" name="row_label">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 700 18pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; text-decoration: underline;&quot;&gt;Row&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="7" column="5" alignment="Qt::AlignHCenter">
     <widget class="QComboBox" name="patrolboat_comboBox">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Segoe UI&quot;;
background-color: rgb(34,139,34);
color: rgb(255, 255, 255);</string>
      </property>
      <item>
       <property name="text">
        <string>Horizontal</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Vertical</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="6" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Maximum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>30</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="destroyer_label">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 700 12pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Destroyer:</string>
      </property>
     </widget>
    </item>
    <item row="8" column="3">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Maximum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>30</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="4">
     <widget class="QLabel" name="column_label">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 700 18pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; text-decoration: underline;&quot;&gt;Column&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="5" column="4">
     <widget class="QSpinBox" name="destroyer_spinBox_2">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(34,139,34);</string>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>10</number>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLabel" name="battleshiplength_label">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 700 10pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;4&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="length_label">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 700 18pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; text-decoration: underline;&quot;&gt;Length&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="carrier_label">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 700 12pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Carrier:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="5" alignment="Qt::AlignHCenter">
     <widget class="QComboBox" name="submarine_comboBox">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Segoe UI&quot;;
background-color: rgb(34,139,34);
color: rgb(255, 255, 255);</string>
      </property>
      <item>
       <property name="text">
        <string>Horizontal</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Vertical</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="submit_pushButton">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>400</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Segoe UI&quot;;
background-color: rgb(34,139,34);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QLabel" name="header_label">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>1</y>
     <width>751</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 600 28pt &quot;Segoe UI Semibold&quot;;
background-color: rgb(147, 211, 251);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Place your battleships&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="playername_label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>390</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 700 18pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Enter Player Name:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="playername_textEdit">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>400</y>
     <width>104</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 700 10-pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
